app/googleanalytics/ga4.py

```diff
@@ -3,72 +3,36 @@
 """
 
 import os
+import uuid
 
-from flask import current_app, Blueprint, request, send_file
-from jsonschema import FormatChecker, ValidationError
-from jsonschema.validators import Draft202012Validator
+from flask import current_app, Blueprint, send_file
 
-from app.googleanalytics.ga4_schemas import ga4_request_schema
 from app.celery.process_ga4_measurement_tasks import post_to_ga4
 
 ga4_blueprint = Blueprint('ga4', __name__, url_prefix='/ga4')
 
-ga4_request_validator = Draft202012Validator(ga4_request_schema, format_checker=FormatChecker(['uuid']))
-
 GA4_PIXEL_TRACKING_IMAGE_PATH = f'{os.getcwd()}/images/ga4_pixel_tracking.png'
 
 
-@ga4_blueprint.route('/open-email-tracking', methods=['GET'])
-def get_ga4():
+@ga4_blueprint.route('/open-email-tracking/<notification>', methods=['GET'])
+def get_ga4(notification):
```

Comment:
Super nit.... should this be `notification_id`?

Reply:
I was going to say no, because its obviously not a full notification, but the other routes use _id so I guess I will